README.md

```diff
@@ -200,20 +193,6 @@ plugins {
 > }
 > ```
 
-Or use buildscript syntax:
```

Comment:
don't think that there are still people who use that, but it takes space in readme